packages/libs/client/src/signing/signWithChainweaver.ts

```diff
@@ -35,24 +28,25 @@ export async function signWithChainweaver<T1 extends string, T2>(
   ...transactions: (IPactCommand & ICommandBuilder<Record<T1, T2>>)[]
 ): Promise<(IPactCommand & ICommandBuilder<Record<T1, T2>>)[]> {
   const quickSignRequest: IChainweaverQuickSignRequestBody = {
-    reqs: transactions.map((t) => {
+    cmdSigDatas: transactions.map((t) => {
```

Comment:
As per KIP-0015, cmdSigDatas should be commandSigDatas

https://github.com/kadena-io/KIPs/blob/master/kip-0015.md#request

Maybe related to this:
https://github.com/kadena-io/chainweaver/issues/797

Reply:
@CryptoPascal31, great catch. We need to change this as well to `commandSigDatas`.

Reply:
Thanks for the review. Let's finish this PR as the latest chainweaver is implemented with `cmdSigDatas` and update the function once chainweaver is released with the updated signing-api.